src/dist/replication/common/replication_common.cpp

```diff
@@ -514,6 +514,9 @@ void replication_options::initialize()
                                          cold_backup_checkpoint_reserve_minutes,
                                          "reserve minutes of cold backup checkpoint");
 
+    bulk_load_provider_root = dsn_config_get_value_string(
```

Comment:
Where is this config used? I don't understand why bulk-load still need a static "root path" for remote file?

Reply:
This is bulk load root path on remote file provider(FDS).

Reply:
So what's the normal value it can be? It's empty by default here.